bodhi/server/templates/master.html

```diff
@@ -38,8 +38,14 @@
     <link href="${request.static_url('bodhi:server/static/css/site.css') }" rel="stylesheet" />
     <link href="${request.static_url('bodhi:server/static/css/datagrepper-feed.css') }" rel="stylesheet" />
     <link href="${request.static_url('bodhi:server/static/css/panel.css') }" rel="stylesheet" />
+    <link href="${request.static_url('bodhi:server/static/selectize/selectize.bootstrap3.css') }" rel="stylesheet" />
     <script src="${request.static_url('bodhi:server/static/js/jquery-1.10.2.min.js')}"></script>
     <script src="${request.static_url('bodhi:server/static/js/Chart-0.2.0.min.js')}"></script>
+    <script type="text/javascript">
+      var settings = {
+          bz_products: ["${ '","'.join(config['bz_products']) | n }"]
+      };
```

Comment:
Probably we could make use of a `bz_baseurl` entry in Bodhi config, so that we don't hardcode the Bugzilla base URL in the two ajax calls in update_form.js

Reply:
added an item to the config to add the BZ rest url, and referenced it in the script. thaznks!